src/v/kafka/server/connection_context.h

```diff
@@ -81,6 +81,7 @@ class connection_context final
     struct session_resources {
         ss::lowres_clock::duration backpressure_delay;
         ss::semaphore_units<> memlocks;
+        ss::semaphore_units<> queue_units;
```

Comment:
i'd add the link to the paper or some comment here.

Reply:
added in commit message